addroom.html

```diff
@@ -27,33 +26,46 @@
             /* vertical-align: middle; */
             /* background-clip:border-box; */
             height:auto;
+            width: 105.08%;
```

Comment:
I am not sure what 105.08% means

Reply:
Thanks for your comment. well, actually, it will not effect the output of the page. Yesterday, I tried to fix the background picture at the left part of screen. But because of the <div class="col-md-6"> can not fill up all the left part of screen and will leave two slices of white edge at both left and right sides. So I decide  to expand the background to fill the white edges. But now, the problem does not exist because I use the background at the bottom of whole screen. By the way, if u think it is necessary, I will set the width as 100%.

Reply:
please set it as 100%, then I have no more questions.

Reply:
I fix it as 100% just now.